### ISSUE TITLE
Error on https://www.furaffinity.net/view/28269072/

### ISSUE THREAD

**Issue by luminescenttrent** [ruffle-rs/ruffle#2539]
# Error Info
Error name: Error
Error message: panicked at 'assertion failed: `(left == right)`
  left: `true`,
 right: `false`: cannot recursively acquire mutex', /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/std/src/sys/wasm/../unsupported/mutex.rs:23:9

# Player Info
Allows script access: false
Player type: Object
SWF URL: //d.facdn.net/art/faerfax/1533760140/1533760140.faerfax_tango_runcolor.swf
Param movie: //d.facdn.net/art/faerfax/1533760140/1533760140.faerfax_tango_runcolor.swf
Param quality: high
Param wmode: direct
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined
Attribute 5: undefined
Attribute 6: undefined

# Page Info
Page URL: https://www.furaffinity.net/view/28269072/
SWF URL: https://d.facdn.net/art/faerfax/1533760140/1533760140.faerfax_tango_runcolor.swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:84.0) Gecko/20100101 Firefox/84.0
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-15
Channel: nightly
Built: 2021-01-15T01:03:45.335Z
Commit: 09a48a71d8608803fd5dba4343c5f10c928f9652

**Comment by cheezeygeek** [comment]
Strange, I wasn't able to reproduce this using the desktop player, the demo page, or the Firefox add-on.

**Comment by GreenReaper** [comment]
Worksforme now they've self-hosted it.

**Comment by Poopooracoocoo** [comment]
Yeah it runs perfectly fine despite the AS3 warning!

**Comment by luminescenttrent** [comment]
Appreciate your feedback! This is still showing up for me on Firefox; previously with add-on, now with the page's built-in Ruffle. I was able to get it working using the desktop player, however.
I do have a lot of extensions on - maybe it's a bad interaction with those? I don't imagine that's Ruffle's problem, but it'll be a pain to troubleshoot myself.
I hope this doesn't make too much trouble for you guys!! you're doing amazing work here, ruffle is the only thing I could find that actually worked to view this even if I did have to use desktop

**Comment by Poopooracoocoo** [comment]
@luminescenttrent Is this error still showing up for you?

**Comment by luminescenttrent** [comment]
> 
> 
> @luminescenttrent Is this error still showing up for you?

Yep. My browser hasn't changed much since I opened this, and an error still appears.
